client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -82,9 +74,9 @@ const Mentions = React.createClass( {
 			showPopover: typeof query === 'string',
 			query,
 		} );
-	},
+	}
```

Comment:
instead of binding these functions we have the option of using the fat-arrow syntax which uses _lexical scope_ instead of _dynamic scoping_ where all the issues come from. this seems to be a pattern many of us are happy with

```js
onKeyUp = ( { keyCode } ) => {
	…
};
```

the same goes for the click handlers, etc… - they do not need to be bound in the constructor (or anywhere) if using this syntax. React lifecycle methods are still outbound for us (render, componentWillReceiveProps, etc…)